providers/base/units/dock/jobs.pxu

```diff
@@ -2355,85 +2354,535 @@ _description: Check dock MAC address pass-through function, please make sure fol
-     4. Did the network works fine after reboot?
+    Mark it Pass if you have rebooted the system.
 
 id: dock/all-hotplug
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after hotplug
+    This test is to check if the device is still working after hotplug.
 _steps:
-     1. Unplug docking station.
-     2. Waiting 30 seconds.
-     3. Plug in docking station.
+    1. Unplug the docking station.
+    2. Wait for 30 seconds.
+    3. Plug in the docking station.
 _verification:
-     1. Did the display works fine after hotplug?
-     2. Did the storage works fine after hotplug?
-     3. Did the audio works fine after hotplug?
-     4. Did the network works fine after hotplug?
+    Mark it Pass if you have hotpluged the docking station.
 
 id: dock/all-suspend
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after suspend
+    This test is to check if the device is still working after suspend.
 _steps:
-     1. Suspend the system.
-     2. Waiting 30 seconds.
-     3. Resume the system.
+    1. Suspend the system.
+    2. Waiting 30 seconds.
+    3. Resume the system.
 _verification:
-     1. Did the display works fine after suspend?
-     2. Did the storage works fine after suspend?
-     3. Did the audio works fine after suspend?
-     4. Did the network works fine after suspend?
+    Mark it Pass if you have suspended and resumed the system.
 
 id: dock/all-poweroff
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after poweroff
+    This test is to check if the device is still working after power-off.
+_steps:
+    1. Poweroff the system.
+    2. Wait for 30 seconds.
+    3. Poweron the system.
+_verification:
+    Mark it Pass if you have powered on and powered off the system.
+
+id: dock/all-init-monitor-multi-head
+category_id: dock-display
+estimated_duration: 120.0
+_summary: Dual monitors test while docked
+plugin: manual
+_purpose:
+    This test is to verify that multi-monitor output works using the dock. You will need multiple external monitors to perform this test. Depends on the specification of the device to use 4K monitor or FHD monitor.
+_steps:
+    Skip this test if your video card or the dock do not support multiple monitors.
+    1. If your dock provides more than one monitor outputs, connect multiple monitors.
+    2. Open the "Displays" tool (open the dash and search for "Displays").
+    3. Configure your output to provide one desktop across all the monitors.
+    4. Open any application and drag its window from one monitor to the next.
+_verification:
+    Was the stretched desktop displayed correctly across all the screens?
+_siblings:
+    [{"id": "dock/all-reboot-monitor-multi-head"},
+     {"id": "dock/all-hotplug-monitor-multi-head"},
+     {"id": "dock/all-susepnd-monitor-multi-head"},
```

Comment:
a typo here and also the rest of suspend cases
I think that's why there aren't any suspend-related tests in your submission

Reply:
@weizhenwu Thanks for the kind reminder!